Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM registry.access.redhat.com/ubi8/ruby-26
-ENV HEIMDALLTOOLS_VERSION 1.3.6
-MAINTAINER rx294@nyu.edu
+FROM ruby:alpine as builder
```

Comment:
The Dockerfile changes should not be showing up in this PR since those are already in master, rebasing this PR should fix the issue.